mem_test.go

```diff
@@ -191,3 +202,28 @@ func BenchmarkHash(b *testing.B) {
 		}
 	}
 }
+
+// very old go like 1.14 doesn't have io.Discard
+type discordWriter struct{}
```

Comment:
[typo?] I believe you intended `discardWriter`?

Reply:
yes, typo